resources-generator/src/main/kotlin/dev/icerock/gradle/utils/StringExt.kt

```diff
@@ -66,12 +66,22 @@ internal val String.appearance: Appearance
 
 internal val String.withoutAppearance: String
     get() {
+        // If name doesn't contains potential suffix - early return
```

Comment:
we should add sample resource for this case

Reply:
We has image without theme suffix, but doesn't have image with short name (less than theme -suffix), and without this changes we has crash. I'm be add image with short name later today